proxy/src/control/discovery.rs

```diff
@@ -369,6 +377,82 @@ where
     }
 }
 
+// ===== impl DestinationSet =====
+
+impl <T: HttpService<ResponseBody = RecvBody>> DestinationSet<T> {
+    fn add<Addrs>(&mut self, authority_for_logging: &FullyQualifiedAuthority, addrs_to_add: Addrs)
+        where Addrs: Iterator<Item = SocketAddr>
```

Comment:
style/tioli:

we tend to use single-letter generics because it helps to differentiate them from concrete types -- seeing `addrs_to_add: A` makes it obvious to me that this is the generic thing.  I have to gather a bit more context to understand that otherwise. I think it's fair to diverge from that when it adds clarity, but not sure it adds much here, where the generic type is only used in the `fn` signature.

I tend to think that a variable name's descriptiveness should reflect its scope: If a variable is only used on the next line, single-character names are fine. If the name is used through a fn definition, it probably should at least be a whole word, and if its used throughout a whole module, it should probably be even more descriptive.